src/main/java/com/cloudcoders/gestaca/logic/course/GetAllCourses.java

```diff
@@ -2,6 +2,7 @@
 
 import com.cloudcoders.gestaca.logic.ICourseDAO;
 import com.cloudcoders.gestaca.model.Course;
+import com.cloudcoders.gestaca.persistance.PersistenceException;
```

Comment:
Well I think that the name is not good, but the use of that exception is more or less good.